### PR TITLE
feat(settings-profile): add functionality for updating about me

### LINKED ISSUE #4595: profile - add functionality for updating about me

### PR DESCRIPTION
**Pull request by nathan-power**
**What this PR does** 📖
- Adds transparent prop to InteractiablesInput
- Adds for updating about me in profile settings

**Which issue(s) this PR fixes** 🔨

Resolve #4595 

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: components/views/navigation/sidebar/list/item/Item.vue [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 <script lang="ts">
 import Vue, { PropType } from 'vue'
-import { mapGetters } from 'vuex'
```
Comment (josephmcg) [inline components/views/navigation/sidebar/list/item/Item.vue:5]: can you add this back? the getters were accidentally removed in that big PR

File: components/ui/ContextMenu/ContextMenu.vue [hunk 2]
```diff
@@ -50,11 +50,12 @@
             :class="{ danger: item.type === 'danger' }"
             @click="(e) => handleAction(e, item.func)"
           >
-            <TypographySubtitle
-              :text="item.text"
+            <TypographyText
               :size="6"
```
Comment (josephmcg) [inline components/ui/ContextMenu/ContextMenu.vue:54]: can remove size prop

**Review by aewing — APPROVED** [review]

File: components/views/settings/pages/profile/Profile.html [hunk 3]
```diff
@@ -80,47 +80,57 @@
...
+                  <InteractablesInput
+                    v-else
+                    v-model="inputs.name"
+                    class="username-input"
+                    :placeholder="$t('user.registration.username_placeholder')"
+                    autofocus
+                    transparent
+                  />
+
+                  <button
```
Comment (josephmcg) [inline components/views/settings/pages/profile/Profile.html:105]: did you intend to use button components with new prop?

**Review by josephmcg — CHANGES_REQUESTED** [review]

File: components/views/settings/pages/profile/Profile.html (hunk 3 above)
Comment (josephmcg) [reply components/views/settings/pages/profile/Profile.html:105]: scratch this, misread

**Review by josephmcg — APPROVED** [review]

**Pull request merged**